test/v3/utils/getPoolStateWithBalancesV3.integration.test.ts

```diff
@@ -61,7 +61,7 @@ describe('add liquidity test', () => {
                         balance: '6273.005585327880043442',
                     },
                 ],
-                totalShares: '6565.147517543863649467',
+                totalShares: '6566.096765428824944083',
```

Comment:
These are "sanity check" values we update after moving block number similar to price impact v3 test?